handlers/get_coverage.go

```diff
@@ -117,13 +125,13 @@ func getCoverage(w http.ResponseWriter, req *http.Request, rc RenderClient, fc F
 	go func() {
 		defer wg.Done()
 		if isNameSearch && q != "" {
-			areas, nsErr = getAreas(pc, ctx, accessToken, filterJob.PopulationType, geogID, q)
+			areas, nsErr = getAreas(cfg, pc, ctx, accessToken, filterJob.PopulationType, geogID, q, currentPg)
```

Comment:
ctx should really always be the first argument when passing around in functions (I know this was already not the case by the time this change came)